apps/condo/domains/user/schema/ChangeUserEmailService.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Generated by `createservice user.ChangeUserEmailService --type mutations`
+ */
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema, getByCondition } = require('@open-condo/keystone/schema')
+
+const { COMMON_ERRORS } = require('@condo/domains/common/constants/errors')
+const { normalizeEmail } = require('@condo/domains/common/utils/mail')
+const access = require('@condo/domains/user/access/ChangeUserEmailService')
+const { INVALID_TOKEN, UNSUPPORTED_TOKEN, TOKEN_NOT_FOUND, USER_NOT_FOUND } = require('@condo/domains/user/constants/errors')
+const { User, UserSudoToken } = require('@condo/domains/user/utils/serverSchema')
+const { changeOrVerifyUserEmailGuard } = require('@condo/domains/user/utils/serverSchema/userEmailGuards')
+const { detectTokenTypeSafely, TOKEN_TYPES } = require('@condo/domains/user/utils/tokens')
+
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    DV_VERSION_MISMATCH: {
+        ...COMMON_ERRORS.DV_VERSION_MISMATCH,
+        mutation: 'changeUserEmail',
+    },
+    WRONG_SENDER_FORMAT: {
+        ...COMMON_ERRORS.WRONG_SENDER_FORMAT,
+        mutation: 'changeUserEmail',
+    },
+    INVALID_TOKEN: {
+        mutation: 'changeUserEmail',
+        variable: ['data', 'token'],
+        code: BAD_USER_INPUT,
+        type: INVALID_TOKEN,
+        message: 'Invalid token',
+        messageForUser: 'api.user.changeUserEmail.INVALID_TOKEN',
+    },
+    UNSUPPORTED_TOKEN: {
+        mutation: 'changeUserEmail',
+        variable: ['data', 'token'],
+        code: BAD_USER_INPUT,
+        type: UNSUPPORTED_TOKEN,
+        message: 'Unsupported token',
+        messageForUser: 'api.user.changeUserEmail.UNSUPPORTED_TOKEN',
+    },
+    TOKEN_NOT_FOUND: {
+        mutation: 'changeUserEmail',
+        variable: ['data', 'token'],
+        code: BAD_USER_INPUT,
+        type: TOKEN_NOT_FOUND,
+        message: 'Unable to find non-expired UserSudoToken by specified token',
+        messageForUser: 'api.user.changeUserEmail.TOKEN_NOT_FOUND',
+    },
+    USER_NOT_FOUND: {
+        mutation: 'changeUserPassword',
+        code: BAD_USER_INPUT,
+        type: USER_NOT_FOUND,
+        message: 'User not found',
+        messageForUser: 'api.user.changeUserEmail.USER_NOT_FOUND',
+    },
+    WRONG_EMAIL_FORMAT:  {
+        ...COMMON_ERRORS.WRONG_EMAIL_FORMAT,
+        variable: ['data', 'newEmail'],
+        mutation: 'changeUserEmail',
+    },
+    OPERATION_FAILED: {
```

Comment:
sounds too generic

Reply:
yes, that's right. Because in some cases you want to hide the real reason for the error